bin/postinstall

```diff
@@ -6,6 +6,21 @@ set -e
 
 CLI="${APP_NAME}"
 
+# Check for custom gems
+custom_gemfile=$(${CLI} config:get CUSTOM_PLUGIN_GEMFILE || echo "")
+if [ -e "${custom_gemfile}"]; then
+
+	# Need to override the frozen setting for the vendored gems
+	${CLI} run bundle config --local frozen 0
+
+	# Re-bundle the application including gems
+	${CLI} run bundle install
```

Comment:
Is there a difference with doing a `bundle install --no-deployment` (and without the `frozen 0` line above)?